SPECS/mesa/mesa.spec

```diff
@@ -106,6 +106,7 @@ BuildRequires:  pkgconfig(xxf86vm)
 BuildRequires:  pkgconfig(zlib) >= 1.2.3
 BuildRequires:  python3-devel
 BuildRequires:  python3-mako
+AutoReq:        no
```

Comment:
Mariner 2.0 toolkit is behaving slightly different than 1.0. Even if a dependency is added for BuildRequires it is pulling by default to Requires. To avoid this we need to add AutoReq to no.